tests/unit/testFiles/inlineMetadata.md

```diff
@@ -0,0 +1,12 @@
+---
+description: some description
+footer: footer content
+slide: title-content
+presenter: presenter name
+---
+# Cover Slide ::slide:cover ::toc:false
+---
+# Section Slide ::slide:section
+---
+# Title Content Slide
+some content
```

Comment:
more (not-happy-path) cases to test:

`# Title with colon slide:notavalue ::slide:cover ::toc:false`

`## Cover Slide ::slide:cover ::toc:false`

`# Cover Slide ::slide:cover ::toc:false ::toc:true`

```
# Cover Slide ::slide:cover ::toc:false
# Cover Slide ::slide:cover ::toc:true
```

Reply:
I have added the cases for,
`# Title with colon slide:notavalue ::slide:cover ::toc:false`
`# Cover Slide ::slide:cover ::toc:false ::toc:true`

in PR https://github.com/JankariTech/revealjs-awesoMD/pull/8. The second case seems to be for heading related and I will cover that in this PR.

I think let's first complete the PR https://github.com/JankariTech/revealjs-awesoMD/pull/8, which is for extracting metadata with markdown file which contains slide separator and then to this PR which adds the slide separator if no slide separator is provided.

I am first working on extracting metadata only to make things less complex.